### DISCUSSION TITLE
Show popover and ellipsis when the text is too long.

### DISCUSSION THREAD

**Discussion by cdwmhcc** [directus/directus#9081] [Feature Requests]
![image](https://user-images.githubusercontent.com/8030134/138508228-249744df-cc6d-4fec-984f-6a4a2f07f142.png)

When the text is too long, the display is abnormal and it looks uncomfortable. I hope to add  ellipsis to display.Like this:

![image](https://user-images.githubusercontent.com/8030134/138508451-3ea2a640-5a78-49c0-8713-1bd6972028b2.png)

In addition, it would be better if you add a popover,like this:

![image](https://user-images.githubusercontent.com/8030134/138508701-95b0d1a8-9b7f-400a-9ca5-a23070847abb.png)

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
